arviz/plots/forestplot.py

```diff
@@ -36,7 +36,6 @@ def plot_forest(
     ax=None,
     backend=None,
     backend_config=None,
-    backend_kwargs=None,
```

Comment:
this argument should not be removed. Also, there is no argument being added, the code examples use `alternate_row_shading` so it should be added here (and documented in the docstring)